scripts/cmake/Find.cmake

```diff
@@ -61,7 +61,7 @@ if(OGS_BUILD_GUI)
         vtkIOImage vtkIOLegacy vtkIOExport vtkIOExportPDF
         vtkIOExportOpenGL2 vtkInteractionStyle vtkInteractionWidgets
         vtkGUISupportQt vtkRenderingOpenGL2 vtkRenderingContextOpenGL2
-        vtkFiltersTexture vtkRenderingCore
+        vtkFiltersTexture vtkRenderingCore vtkFiltersParallel
```

Comment:
@endJunction This part here is why I added OGS_USE_GUI for the test.

Reply:
@endJunction In the cli-only case we just require a very small subset of VTK (not containing any filters).